scanspec/cli.py

```diff
@@ -48,3 +48,11 @@ def service(cors):
     from scanspec.service import run_app
 
     run_app(cors)
+
+
+@cli.command()
+def schema():
```

Comment:
Can we have a test for this please? Doesn't need to be very extensive, just check it prints something that is valid JSON with a "$schema" key

Reply:
Here's some docs: https://click.palletsprojects.com/en/7.x/testing/

Reply:
Unfortunately the schema is not stored as JSON but in the GraphQL Schema Definition Language (SDL). So we've added a couple of tests to first make sure something is printed, and then make sure that the thing that was printed is of type GraphQLSchema.